hvpy/facade.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+
+from hvpy.core import execute_api_call
+from hvpy.parameters import getJP2ImageInputParameters
+
+__all__ = ["getJP2Image"]
+
+
+def _copy_docstring(input_class):
+    def decorator(func):
+        docstring = input_class.__doc__.split("\n")[1:]
+        docstring2 = func.__doc__.split("\n")[1:]
+        docstring[0] = docstring2[0]
+        docstring = "\n".join(docstring)
+        func.__doc__ = docstring
+        return func
+
+    return decorator
+
+
+@_copy_docstring(getJP2ImageInputParameters)
+def getJP2Image(
+    *,
+    date: datetime,
+    sourceId: int,
+    jpip: bool = False,
+    json: bool = False,
+):
+    """
+    Get a JP2 image from the helioviewer.org API.
+    """
+    params = locals()
+    params = getJP2ImageInputParameters(**params)
```

Comment:
Why can't we just pass in the values from the function into `getJP2ImageInputParameters`?

Reply:
As I understand, we need to pass a dictionary to the Pydantic baseModel
like this example: https://pydantic-docs.helpmanual.io/#example

And to make a dictionary of params, locals() is a clean way to do it.
https://github.com/Helioviewer-Project/python-api/pull/33#discussion_r911891689

Reply:
What happens if you try to pass them in as keyword arguments?

Reply:
It throws an error, 
TypeError: __init__() takes exactly 1 positional argument (2 given)

Reply:
How are you passing them into the class?

Reply:
Oh, sorry I directly passed the locals as keyword arg.
Done.